server/src/test/java/org/elasticsearch/cluster/coordination/CoordinatorTests.java

```diff
@@ -1101,6 +1102,9 @@ private static int defaultInt(Setting<Integer> setting) {
         private final Set<String> blackholedNodes = new HashSet<>();
         private final Map<Long, ClusterState> committedStatesByVersion = new HashMap<>();
 
+        private final Function<DiscoveryNode, PersistedState> persistedStateSupplier = localNode -> new MockPersistedState(0L,
```

Comment:
can you call this `defaultPersistedStateSupplier`?

Reply:
sure